Gemfile.lock

```diff
@@ -39,56 +42,57 @@ GEM
       diff-lcs (>= 1.1.3, < 2.0)
     rspec-mocks (2.13.1)
     sequel (4.9.0)
-    simplecov (0.7.1)
+    simplecov (0.9.2)
+      docile (~> 1.1.0)
       multi_json (~> 1.0)
-      simplecov-html (~> 0.7.1)
-    simplecov-html (0.7.1)
-    sinatra (1.4.4)
+      simplecov-html (~> 0.9.0)
+    simplecov-html (0.9.0)
+    sinatra (1.4.6)
       rack (~> 1.4)
       rack-protection (~> 1.4)
-      tilt (~> 1.3, >= 1.3.4)
-    text (1.2.3)
+      tilt (>= 1.3, < 3)
+    text (1.3.0)
     thor (0.19.1)
-    tilt (1.4.1)
-    timecop (0.6.3)
-    torquebox (3.1.1)
+    tilt (2.0.1)
+    timecop (0.7.3)
+    torquebox (3.1.2)
```

Comment:
No. It's going to get mad at you for not having the right version available, since we have 3.1.1 on the builders. We need to build and ship pl-jruby to have on our builders, as well as razor-torquebox to pull into the mock during the build process